packs/elastic/README.md

```diff
@@ -0,0 +1,170 @@
+# Elasticsearch integration pack
+
+Pack provides many operations helping to manage your Elasticsearch indices. Current functionality is mostly based on [curator](http://www.elastic.co/guide/en/elasticsearch/client/curator/current/).
+
+## Features
+
+Pack provides a set of actions based on curator.
+
+## Curator based actions
+
+Action | Description
+------ | -----------
+**alias** | Add indices to or remove them from aliases.
+**allocation** | Set routing allocation based on tags.
+**bloom** | Disable the bloom filter cache for indices.
+**close** | Close indices.
+**delete.indices** | Delete indices.
+**delete.snapshots** | Delete snapshots.
+**open** | Open indices.
```

Comment:
I agree explicit way is good. I've taken the naming from curator which the pack is based on. Well this naming seems sane, *delete.**, *show.** - are explicit. Actions like,*open*, *alias*, *optimize* make sense only for indices, so I guess regrouping is not really needed or really helps with anything.

> This way actions which operate on the same resource are organized / grouped together ...
The are grouped by action operation rather than by the same resource. It's more likely an operation brings a similar interface for actions, rather than grouping by the resource (indices or snapshots)...